index.html

```diff
@@ -220,6 +219,75 @@ <h3>Skills</h3>
 			</div>
 		</section>
 
-		<script src="main.js"></script>
-	</body>
-</html>
+        <section id="contact">
```

Comment:
- [ ] As mentioned by the previous reviewers, there is an overlapping issue between skills and contact section which needs to be fixed in order to proceed. kindly consider fixing it. the following image of your design is at `1024px`


your design
![qwer](https://user-images.githubusercontent.com/96683843/213876585-83c1f50e-7d07-4818-b90b-ec7b3be94375.jpg)